hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/BaseContainerManagerTest.java

```diff
@@ -541,4 +541,6 @@ public static ContainerId createContainerId(int cId, int aId) {
         ContainerId.newContainerId(appAttemptId, cId);
     return containerId;
   }
+
```

Comment:
Revert this file since the changes are a no-op?

Reply:
I will fix this issue.